src/transformers/modeling_utils.py

```diff
@@ -1982,6 +1992,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             elif not low_cpu_mem_usage:
                 raise ValueError("Passing along a `device_map` requires `low_cpu_mem_usage=True`")
 
+        if keep_in_fp32_modules is not None and not low_cpu_mem_usage:
+            # Force `low_cpu_mem_usage` to be set to `True` - check the PR:
+            logger.warning(
+                "The argument `keep_in_fp32_modules` is used, force-enabling `low_cpu_mem_usage` to load the model"
+            )
+            low_cpu_mem_usage = True
```

Comment:
Shouldn't be force-set here.

Reply:
proposed something in [2bd00b3](https://github.com/huggingface/transformers/pull/20683/commits/2bd00b3eaf1316cecbc76b9ef281b9d8f8a3e26d)